src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModule.kt

```diff
@@ -43,11 +43,15 @@ class SwerveModule(val io: SwerveModuleIO) {
 
   var modulePosition = SwerveModulePosition()
 
+  var positionDeltas: Array<SwerveModulePosition?>
```

Comment:
again this doesn't need to be here but get rid of the `?` and initialize in declaration by doing `var positionDeltas: Array<SwerveModulePosition> = arrayOf();` or something like that